server/lib/src/be/mod.rs

```diff
@@ -872,10 +873,13 @@ pub trait BackendTransaction {
             .get_db_ts_max()
             .and_then(|u| u.ok_or(OperationError::InvalidDbState))?;
 
-        let bak = DbBackup::V2 {
+        let keyhandles = idlayer.get_key_handles()?;
+
+        let bak = DbBackup::V3 {
             db_s_uuid,
             db_d_uuid,
             db_ts_max,
+            keyhandles,
```

Comment:
Can we call it something more ... clear? I know *you* know what it means and I know what it means in this very specific context, but ... `keyhandles` ... is confusing.

Maybe `replication_keys`? There's a lot of keys in this system, and I hope we're not stuffing them all in one place 😄

Reply:
The reason it's keyhandles is that in the future I plan to use this for persistent storage of keyhandles that are loaded into a tpm. So keyhandles is meant to be "here is where we persist key material". 

Which btw, is getting a bit into how a TPM works, but they don't store anything internally. You have to temporarily load them in and out, and we have to store the encrypted blobs ourselves. 

Anyway, that's why it's keyhandles.

Reply:
Ok, cool, thx 😄